File: src/main/java/com/natia/secretmod/hooks/ItemPickupHook.java
package com.natia.secretmod.hooks;

import com.natia.secretmod.core.ItemPickupEvent;
import com.natia.secretmod.features.AlertPickups;
import com.natia.secretmod.features.fishing.WormCounter;
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;

public class ItemPickupHook {
    @SubscribeEvent
    public void onPickup(ItemPickupEvent event) {
        WormCounter.getInstance().onCollect(event);
        AlertPickups.getInstance().pickup(event);
    }
}

File: src/main/kotlin/natia/skytweaks/hooks/ItemPickupHook.kt
package natia.skytweaks.hooks

import natia.skytweaks.core.ItemPickupEvent
import natia.skytweaks.features.AlertPickups
import natia.skytweaks.features.fishing.WormCounter
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent

class ItemPickupHook {
    @SubscribeEvent
    fun onPickup(event: ItemPickupEvent) {
        WormCounter.instance.onCollect(event)
        AlertPickups.instance.pickup(event)
    }
}
